assets/css/components/_news-and-views.scss

```diff
@@ -0,0 +1,37 @@
+figure {
```

Comment:
According to the wireframe, preview images should not be shown on the news/views pages of the mobile design.

Reply:
In the wireframe, preview images are not shown in the desktop design either. Can we discuss this later today?

Reply:
As we talk earlier, the preview image should not be shown on the mobile design.

Reply:
The format is a little different for News and Views. News tiles do not have images. Since News posts are not authored by us and link to external sites it will either require a lot of work to source images for posts or it won't look great if use placeholder. For Views pages, there is an image in desktop view, but it is not visible on mobile.

Reply:
Right. @TeddTech, please hide the preview image on the mobile design.

Reply:
@greatislander Was only able to get this to work by using `!important`.  I know it is not recommended to use this so lets discuss work arounds during todays We Count check in.